File: ntss-win/FNWConverter/convert-server-side/src/main/java/batch/entity/MstBed.java
package batch.entity;

import java.sql.Timestamp;
import org.seasar.doma.Entity;
import org.seasar.doma.Id;
import org.seasar.doma.Table;
import org.seasar.doma.jdbc.entity.NamingType;

import lombok.Getter;
import lombok.Setter;


/**
 * mst_bed(ベッドマスタ)のエンティティクラス
 */
@Entity(naming = NamingType.SNAKE_LOWER_CASE)
@Table(name = "mst_bed")
@Getter
@Setter
public class MstBed extends BaseBlankEntity {
  @Id
  /**
   * ベッドコード
   */
  private Long bedCd;

  /**
   * 施設コード
   */
  private String facilityCd;

  /**
   * FNW+で管理する施設内の一意なベッド番号
   */
  private Double fnBedNo;

  /**
   * ベッド番号
   */
  private Integer bedNo;

  /**
   * ベッド名
   */
  private String bedName;

  /**
   * シャント位置
   */
  // add FNSI-分類不一致判断の追加 徐 start
  private Short shuntPosition;
  // add FNSI-分類不一致判断の追加 徐 end
  /**
   * 感染症フラグ
   */
  private String isInfection;

  /**
   * 緊急区分
   */
  private Double emergencyClass;

  /**
   * 装置番号
   */
  private Long machineNo;

  /**
   * 出力先プリンタ名
   */
  private String outputPrinter;

  /**
   * 前体重測定時の自動印刷有無
   */
  private String isAutoprintBefore;

  /**
   * 後体重測定時の自動印刷有無
   */
  private String isAutoprintAfter;

  /**
   * 実績確定時の自動印刷有無
   */
  private String isAutoprintCommit;
  /**
   * 連携コード1
   */
  private String inHospitalCd_1;
  /**
   * 連携コード2
   */
  private String inHospitalCd_2;
  /**
   * 表示フラグ
   */
  private String isDisp;

  /**
   * 削除フラグ
   */
  private String isDel;

  /**
   * 登録日時
   */
  private Timestamp regDate;

  /**
   * 更新日時
   */
  private Timestamp upDate;

  /**
   * 在宅フラグ
   */
  private String isHomeDialysis;
}

File: Source/ntss-win/NKKWeight/Source/NKKWeightScaleDB/Models/Mst_bed.cs
namespace NKKWeightScaleDB.Models
{
    using Newtonsoft.Json;

    public class Mst_bed : BaseEntity
    {
        [JsonProperty(PropertyName = "bedCd")]
        public string bed_cd { get; set; }

        [JsonProperty(PropertyName = "facilityCd")]
        public string facility_cd { get; set; }

        [JsonProperty(PropertyName = "bedNo")]
        public string bed_no { get; set; }

        [JsonProperty(PropertyName = "bedName")]
        public string bed_name { get; set; }

        [JsonProperty(PropertyName = "shuntPosition")]
        public string shunt_position { get; set; }

        [JsonProperty(PropertyName = "isInfection")]
        public string is_infection { get; set; }

        [JsonProperty(PropertyName = "emergencyClass")]
        public string emergency_class { get; set; }

        [JsonProperty(PropertyName = "machineNo")]
        public string machine_no { get; set; }

        [JsonProperty(PropertyName = "outputPrinter")]
        public string output_printer { get; set; }

        [JsonProperty(PropertyName = "isAutoprintBefore")]
        public string is_autoprint_before { get; set; }

        [JsonProperty(PropertyName = "isAutoprintAfter")]
        public string is_autoprint_after { get; set; }

        [JsonProperty(PropertyName = "isAutoprintCommit")]
        public string is_autoprint_commit { get; set; }

        [JsonProperty(PropertyName = "fnBedNo")]
        public string fn_bed_no { get; set; }

        [JsonProperty(PropertyName = "isDisp")]
        public string is_disp { get; set; }

        [JsonProperty(PropertyName = "isDel")]
        public string is_del { get; set; }

        [JsonProperty(PropertyName = "regDate")]
        public string reg_date { get; set; }

        [JsonProperty(PropertyName = "upDate")]
        public string up_date { get; set; }

        [JsonProperty(PropertyName = "isHomeDialysis")]
        public string is_home_dialysis { get; set; }
    }
}